### PR TITLE
Add a sanity test for beta releases

### PR DESCRIPTION
**Pull request by ilya-zuyev**
This PR extends `make check-release` target to validate beta releases along with GA

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -637,7 +637,7 @@ release-hyperkit-driver: install-hyperkit-driver checksum ## Copy hyperkit using
 
 .PHONY: check-release
 check-release: ## Execute go test
-	go test -v ./deploy/minikube/release_sanity_test.go -tags=release
+	go test -timeout 42m -v ./deploy/minikube/release_sanity_test.go
```
Comment (medyagh) [inline Makefile:640]: why is the timeout changed to 42minutes?

File: deploy/minikube/releases-beta.json [hunk 2]
```diff
@@ -18,9 +18,9 @@
   {
       "name": "v1.20.0-beta.0",
       "checksums": {
-          "darwin": "",
-          "linux": "",
-          "windows": ""
+          "darwin": "686f8d7c06c93f28543f982ec56a68544ab2ad6c7f70b39ede5174d7bac29651",
```
Comment (medyagh) [inline deploy/minikube/releases-beta.json:21]: is this change related to this PR?

Comment (ilya-zuyev) [reply deploy/minikube/releases-beta.json:21]: yes, with empty hashes the test for beta versions would fail

File: Makefile (hunk 1 above)
Comment (ilya-zuyev) [reply Makefile:640]: The test downloads all the releases, it takes some time. Now we're going to check beta releases too and it's going to be event longer. The default timeout was 10 minutes and it wasn't enough.
